test fails

devel branch, aretomo2 1.1.2
```
[   FAILED ] TestAreTomo.test_align_02


Traceback (most recent call last):
File "/public/EM/Scipion/miniconda3/envs/scipion3/lib/python3.8/unittest/case.py", line 60, in testPartExecutor
yield
File "/public/EM/Scipion/miniconda3/envs/scipion3/lib/python3.8/unittest/case.py", line 676, in run
self._callTestMethod(testMethod)
File "/public/EM/Scipion/miniconda3/envs/scipion3/lib/python3.8/unittest/case.py", line 633, in _callTestMethod
method()
File "/public/EM/Scipion/scipion-dev/scipion-em-plugins/scipion-em-aretomo/aretomo/tests/test_protocols_aretomo.py", line 191, in test_align_02
self.checkTiltSeries(getattr(prot, OUT_TS, None),
File "/public/EM/Scipion/scipion-dev/scipion-em-plugins/scipion-em-tomo/tomo/tests/test_base_centralized_layer.py", line 198, in checkTiltSeries
self.checkObjectEnabled(ti, isExcludedView, tsId, ind)
File "/public/EM/Scipion/scipion-dev/scipion-em-plugins/scipion-em-tomo/tomo/tests/test_base_centralized_layer.py", line 799, in checkObjectEnabled
self.assertFalse(enb, msg='TsId = %s: %s %i was expected not to be Enabled' % (tsId, objType, ind))
File "/public/EM/Scipion/miniconda3/envs/scipion3/lib/python3.8/unittest/case.py", line 759, in assertFalse
raise self.failureException(msg)
AssertionError: True is not false : TsId = TS_54: Tilt image 0 was expected not to be Enabled
```

--- a/CHANGES.txt
+++ b/CHANGES.txt
@@ -1,3 +1,4 @@
+3.9.4: set aretomo2 1.0.0 as default, because 1.1.2 has broken remove dark option
 3.9.3: hotfix: _defineVariables method must be a class method to be compatible with scipion command config -p
 3.9.2:
    - Aretomo does nos fail when closing its output when process is killed

--- a/aretomo/__init__.py
+++ b/aretomo/__init__.py
@@ -33,7 +33,7 @@ from pyworkflow import VarTypes
 from .constants import *
 
 
-__version__ = '3.9.3'
+__version__ = '3.9.4'
 _logo = "aretomo_logo.png"
 _references = ['Zheng2022']
 
@@ -46,8 +46,9 @@ class Plugin(pwem.Plugin):
 
     @classmethod
     def _defineVariables(cls):
-        cls._defineEmVar(ARETOMO_HOME, f'aretomo2-{V1_1_2}',
-                         description="Root folder where aretomo was extracted. Is assumes binaries are under that folder/bin.",
+        cls._defineEmVar(ARETOMO_HOME, f'aretomo2-{V1_0_0}',
+                         description="Root folder where aretomo was extracted. Is assumes "
+                                     "binaries are under that folder/bin.",
                          var_type=VarTypes.FOLDER)
         cls._defineVar(ARETOMO_CUDA_LIB, pwem.Config.CUDA_LIB,
                        description="Path to the CUDA lib path to use with the Aretomo binary.",
@@ -64,7 +65,8 @@ class Plugin(pwem.Plugin):
             binaryName = f'AreTomo2_{binaryStr}_Cuda{cudaVersion.major}{cudaVersion.minor}'
 
         cls._defineVar(ARETOMO_BIN, binaryName,
-                       description="Aretomo binary file to use. Should match the cuda pointed by %s. It also should be under %s/bin folder" % (ARETOMO_CUDA_LIB, ARETOMO_HOME),
+                       description="Aretomo binary file to use. Should match the cuda pointed by %s. "
+                                   "It also should be under %s/bin folder" % (ARETOMO_CUDA_LIB, ARETOMO_HOME),
                        var_type=VarTypes.FILENAME)
 
     @classmethod

